src/shared/helpers/formControl.js

```diff
@@ -22,7 +22,45 @@ const getIconButton = (label, buttonType, iconType, iconSize, onClick) => {
     testId={`cf-ui-button-${label}`}/>;
 };
 
+
+
+const getTextField = (
+  textValue, 
+  onChange, 
+  validationMessage, 
+  { 
+    id = 'input1', 
+    name = id, 
+    labelText = 'label1', 
+    helpText = '', 
+    required = false, 
+    requiredText = required ? 'required' : '', 
+    textInputProps = { 
+      disabled: false, 
+      placeholder: labelText, 
+      type: 'text' 
```

Comment:
We should pass this field type in as a param.   See https://f36.contentful.com/components/text-input/#textinput-type.

This should help with validation.    Sorry if I missed this in the code, but seems hardcoded right now

Reply:
It's not hardcoded, it is a default param, in case someone just wants a simple text field, they won't have to pass that in